scripts/build/store-azp-variables.ps1

```diff
@@ -0,0 +1,17 @@
+# Read the project version from the Version.props file
+$VersionFilePath = "$env:BUILD_SOURCESDIRECTORY\scripts\version\Version.props"
+Write-Host "Reading the Sonar project version from '${VersionFilePath}' ..."
+
+[xml]$VersionProps = Get-Content "$VersionFilePath"
+$Version = $VersionProps.Project.PropertyGroup.MainVersion
+Write-Host "Sonar project version is '${Version}'"
+
+# Save variables to files so they can be used by other tasks and stages
+# -Force to suppress the error message when it exists on reused agent
+# | Out-Null to suppress noisy output report
```

Comment:
```suggestion
# - Out-Null to suppress noisy output report
```

Reply:
This was intentional as a code fragment for output redirection, not a bullet point. Do you want to change it?